grow/templates/filters.py

```diff
@@ -42,6 +42,11 @@ def deeptrans(ctx, obj):
     """Deep translate an object."""
     return _deep_gettext(ctx, obj)
 
+@jinja2.contextfilter
+def expand_partial(_ctx, partial_name):
+    """Filter for expanding partial path from name of partial."""
+    return '/partials/{0}/{0}.html'.format(partial_name)
```

Comment:
hmm... can you always assume there are two things to substitute in `partial_name`?

Everything else looks good. :)

Reply:
It is using positional substitution so it comes out to be `/partials/name/name.html`